tsdb/db_test.go

```diff
@@ -2668,3 +2668,28 @@ func TestChunkWriter(t *testing.T) {
 		})
 	}
 }
+
+func TestRangeForTimestamp(t *testing.T) {
+	type args struct {
+		t     int64
+		width int64
+	}
+	tests := []struct {
+		name     string
```

Comment:
You don't need this, you can use the first value of the range.

Reply:
fixed.